src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -34,13 +34,13 @@
     <Reference Include="System.Runtime.InteropServices" />
     <Reference Include="System.ComponentModel" />
     <Reference Include="System.ComponentModel.Primitives" />
+    <Reference Include="System.ComponentModel.TypeConverter" />
```

Comment:
We actually don't want this dependency. It was previously there and we removed it due to layering concerns.

Reply:
And you are correct that we have duplicate `InvalidEnumArgumentException` types, but that is going to be okay for now. The one defined in System.Drawing.Common is internal, and the real type is being moved down into System.ComponentModel.Primitives, but that won't be available until .NET Core 2.1. In the meantime, we are building against .NET Core 2.0 and will just include an internal version of the exception.

Reply:
OK, I'll revert the change. It causes one unit test to fail, I'll disable that test for now.